src/compiler/checker.ts

```diff
@@ -1093,38 +1093,71 @@ namespace ts {
             return links.resolvedExports || (links.resolvedExports = getExportsForModule(moduleSymbol));
         }
 
-        function extendExportSymbols(target: SymbolTable, source: SymbolTable) {
+        interface ExportedSymbolDiagnostics {

```

Comment:
It's not clear what this is; "diagnostics" typically implies something to do with error messages.